### ISSUE TITLE
New translation: Serbian

### ISSUE THREAD

**Issue by taocnrut** [squidfunk/mkdocs-material#835]
## Instructions

1. Check, if your language is already available: [here](http://bit.ly/2DCzaL0)
2. If it isn't, please translate the labels on the right:

``` jinja
{% macro t(key) %}{{ {
  "language": "en",
  "clipboard.copy": "Копирај у клипборд",
  "clipboard.copied": "Ископирано у клипборд",
  "edit.link.title": "Уреди страницу",
  "footer.previous": "Претходно",
  "footer.next": "Следеће",
  "meta.comments": "Коментари",
  "meta.source": "Извор",
  "search.language": "sr",
  "search.placeholder": "Претрага",
  "search.result.placeholder": "Унесите појам претраге",
  "search.result.none": "Нису пронађени документи",
  "search.result.one": "1 резултат претраге",
  "search.result.other": "# резултата претраге",
  "search.tokenizer": "[\s\-]+",
  "skip.link.title": "Иди на текст",
  "source.link.title": "Иди у репозиторијум",
  "toc.title": "Садржај"
}[key] }}{% endmacro %}
```

Thanks!

**Comment by squidfunk** [comment]
Thank you for your contribution! Added both in fd75fa8eb71f1ffa8282b36dced8948de00fc0fa and will be part of the next release.

**Comment by squidfunk** [comment]
Released as part of 3.0.0.
